docs/configuration.md

```diff
@@ -172,6 +173,7 @@ Mandatory parameters are marked as **Required**, which means that they are requi
 | `user_data_dir` | Directory containing user data. <br> *Defaults to `./user_data/`*. <br> **Datatype:** String
 | `dataformat_ohlcv` | Data format to use to store historical candle (OHLCV) data. <br> *Defaults to `json`*. <br> **Datatype:** String
 | `dataformat_trades` | Data format to use to store historical trades data. <br> *Defaults to `jsongz`*. <br> **Datatype:** String
+| `position_adjustment_enable` | Enables the strategy to use position adjustments (additional buys or sells). More information below. <br> **Datatype:** Boolean
```

Comment:
I think this parameter should become a strategy parameter (set / settable in the strategy).

It's almost exclusively dependent on the strategy, not on the configuration - forcing people to adjust this in the config is not a good approach if you ask me.

This will be as simple as 
* adding it to the below list
* adding it to strategy_resolver (next to all the other parameters)
* adding a default value in the strategy.

Reply:
That's a great idea. This keeps this whole thing away from config file at all and all tight and neat inside the strategy.